awx/conf/settings.py

```diff
@@ -187,6 +184,18 @@ def get_settings_to_cache(registry):
     return dict([(key, SETTING_CACHE_NOTSET) for key in get_writeable_settings(registry)])
 
 
+# HACK: runs in thread in order to work in an asyncio context
```

Comment:
Could we disable database settings for the ws_relay command instead? Are there any settings that affect its behavior that need to be dynamic?